app/experimenter/templates/experiments/section_overview.html

```diff
@@ -43,4 +43,13 @@
 
   {% endif %}
 
+  {% if experiment.projects.all %}
+    <p><strong> Projects</strong></p>
+    <p>
+      {% for project in experiment.projects.all %}
+      {{project}}{% if not forloop.last %},{% endif %}
+    {% endfor %}
+    </p>
+  {% endif %}
```

Comment:
We might wanna make these links, but that's a small thing we can do as a follow up